docs/how-to-install.md

```diff
@@ -6,145 +6,151 @@ license: "This software is licensed under the Apache License version 2."
 
 ## Install Apache Gravitino from scratch
 
+### Prerequisites
+
+If you have not installed Java, install it and configure the `JAVA_HOME` environment variable. After installation, you can run the `${JAVA_HOME}/bin/java -version` command to check the Java version.
```

Comment:
The convention is to treat docs as code, and to use git for change tracking.
Since git works on a line-by-line comparison, we'd better wrap the docs manually
so that changes can be easily tracked and reviewed.

```suggestion
If you have not installed Java, install it and configure the `JAVA_HOME` environment variable.
After the installation, you can run the `${JAVA_HOME}/bin/java -version` command to check the Java version.
```

Reply:
Got it. check through the whole doc and fix it. PTAL